brotab/main.py

```diff
@@ -398,6 +422,17 @@ def parse_args(args):
     parser_activate_tab.add_argument('tab_id', type=str, nargs=1,
                                      help='Tab ID to activate')
 
+    parser_activateFocus_tab = subparsers.add_parser(
```

Comment:
@naeloob I don't think more command is necessary. Instead, I'd like to add a `--focus` flag to `activate` command.

Reply:
@naeloob for the record, I'm adjusting your pull request in another pull request here: https://github.com/balta2ar/brotab/pull/22
I didn't want to bother you with the code review so I decided to fix the code according to my taste. I hope it's OK with you.

Reply:
> @naeloob I don't think more command is necessary. Instead, I'd like to add a `--focus` flag to `activate` command.

That's rigth. I didn't know how to add a flag :)

> @naeloob for the record, I'm adjusting your pull request in another pull request here: #22
> I didn't want to bother you with the code review so I decided to fix the code according to my taste. I hope it's OK with you.

No problem friend.